src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+  width: 100%;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid $white;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    align-self: center;
+
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin: 0 0 16px;
+  }
+
+  &__rev {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 24px;
+  }
+
+  &__rev-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-start {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary;
+      margin: 0;
+    }
+
+    &-end {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    display: block;
+    background-color: $blue;
+    color: white;
+    text-decoration: none;
+    border-radius: 5px;
+    padding: 12px 70px 12px 69px;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
```

Comment:
there is no need to set other display, paddings and height -> just use `line-height: 40px` and `align-text: center` 😉